modules/docs/mdx/9.0-UPGRADE-GUIDE.mdx

```diff
@@ -378,6 +381,15 @@ previously used `actionText` or `onActionClick`. The codemod will also update im
 > **Note:** You will manually need to set `mode` to `alert` if your `Toast` conveys urgent and
 > important information such as an error.
 
+---
+
+### Collection
+
+Navigation was updated to use numberical indexes instead of string ids. The `model.state.cursorId`
```

Comment:
```suggestion
Navigation was updated to use numerical indexes instead of string `id`s. The `model.state.cursorId`
```

Reply:
Also, I guess my spell checker doesn't work on MDX files?

Reply:
can't believe you've made me correct your english